setup.py

```diff
@@ -123,7 +123,7 @@
 extras_require = setuptools_args['extras_require'] = {
     ':python_version<"3.3"' : ['ipython>=4.0.0,<6.0.0'],
     ':python_version>="3.3"': ['ipython>=4.0.0'],
-    ':python_version>="3.6"': ['jupyterlab_widgets>=1.0.0'],
+    ':python_version>="3.6"': ['jupyterlab_widgets>=1.0.0,<3'],
```

Comment:
I'm a bit confused - is this saying that anything over python 3.6 should use jupyterlab_widgets 1.x? Should we add a new row now for Python 3.7 or above?
```suggestion
    ':python_version>="3.6"': ['jupyterlab_widgets>=1.0.0,<3'],
    ':python_version>="3.7"': ['jupyterlab_widgets>=3.0.0'],
```

Reply:
I guess that is the way to automatically install `jupyterlab_widgets` when installing `ipywidgets`.

If we dropped support for Python 3.6 on the 7.x branch we could also just replace the existing `:python_version>="3.6"` by `:python_version>="3.7"` directly.

Reply:
> Should we add a new row now for Python 3.7 or above?

Not sure about the new line, I think the pin on `jupyterlab_widgets>=1.0.0,<3` is still relevant on this `7.x` branch to avoid pulling `ipywidgets` 8 and `jupyterlab_widgets` 3 which don't seem compatible.

Reply:
Ah, sorry, I missed that this was for the 7.x branch. In that case, your changes make a lot of sense.